### ISSUE TITLE
ADD . /path/in/docker-image much much slower than before (OSX)

### ISSUE THREAD

**Issue by 34code** [docker/toolbox#481]
There were initially speed problems with using intel adapters in virtualbox (v/s paravirtual adapter)

Now there is still a problem with `ADD` instructions running REALLY slow..

My build times have changed from ~5 min to ~55 min (about an order of magnitude increase)..

I've been looking into aufs vs btrfs and such but no luck. I'm running El Capitan and using the latest 1.11 version of the toolbox release..

**Comment by thaJeztah** [comment]
Would you be interested in testing Docker for Mac? https://blog.docker.com/2016/03/docker-for-mac-windows-beta/

**Comment by 34code** [comment]
absolutely! I'm currently resorting to running docker in a virtual machine for builds. Having a native working version would be ideal. happy to test :)

**Comment by 34code** [comment]
signed up
